### ISSUE TITLE
module 'ds_ctcdecoder' has no attribute 'DS_ERR_SCORER_NO_TRIE'

### ISSUE THREAD

**Issue by GaurangTandon** [mozilla/DeepSpeech#3020]
- I'm following the "Build your own scorer" section of [the docs](https://deepspeech.readthedocs.io/en/master/Scorer.html). I have not modified code in the repo.
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 18.0.4
- **TensorFlow installed from (our builds, or upstream TensorFlow)**: basically did `pip install --upgrade --force-reinstall -e .` mentioned on [TRAINING docs](https://deepspeech.readthedocs.io/en/v0.7.1/TRAINING.html), which installed tensorflow in my virtual environment.
- **TensorFlow version (use command below)**: v1.15.0-92-g5d80e1e 1.15.2
- **Python version**: 3.7.3
- **Exact command to reproduce**:

```bash
$ virtualenv venv
$ source venv/bin/activate
$ cd ~/DeepSpeech
$ pip install --upgrade pip==20.0.2 wheel==0.34.2 setuptools==46.1.3
$ pip install --upgrade --force-reinstall -e .
$ cd native_client/ctcdecode
$ make bindings NUM_PROCESSES=8
$ pip install dist/*.whl
$ cd ../python
$ make bindings
$ pip install dist/deepspeech*
$ cd ~/DeepSpeech/data/lm
$ python generate_lm.py --input_txt path/to/input.transcript --output_dir . --kenlm_bins ~/path/to/kenlm/build/bin --arpa_order 4 --max_arpa_memory "90%" --binary_a_bits 255 --binary_q_bits 8 --binary_type trie --top_k 50000 --arpa_prune "0" --discount_fallback
$ python generate_package.py --alphabet ../alphabet.txt --lm lm.binary --vocab vocab-50000.txt --package kenlm.scorer --default_alpha 1.234 --default_beta 1.012
```

here the last three commands are from [Building your own scorer](https://deepspeech.readthedocs.io/en/master/Scorer.html). The others are from the README in [native_client directory](https://github.com/mozilla/DeepSpeech/tree/master/native_client).

This is the traceback error from the last command:

```
73 unique words read from vocabulary file.
Doesn't look like a character based model.
Using detected UTF-8 mode: False
Traceback (most recent call last):
  File "generate_package.py", line 153, in <module>
    main()
  File "generate_package.py", line 148, in main
    args.default_beta,
  File "generate_package.py", line 58, in create_bundle
    if err != ds_ctcdecoder.DS_ERR_SCORER_NO_TRIE:
AttributeError: module 'ds_ctcdecoder' has no attribute 'DS_ERR_SCORER_NO_TRIE'
```

---

**Note:** Earlier I had tried building everything from source, but I ran into [this error](https://github.com/mozilla/DeepSpeech/issues/2973), so I only focused on installing the python bindings and ctc decoder package.

**Comment by reuben** [comment]
> $ pip install dist/*.whl

Did this actually install the package? You didn't share logs. Try with `-U` to force the upgrade.

**Comment by GaurangTandon** [comment]
Thanks for the quick response @reuben ! I just created a new virtual environment, and added a `-U` while doing the pip install and it worked!

**Comment by Duxy1996** [comment]
I had the same issue. I solved it changing to the stable branch 0.7.1 such is said [here](https://discourse.mozilla.org/t/failed-to-creating-scorer-model-ds-err-scorer-no-trie-solved/60897/6). Before that I was working on the master branch.
